x/group/simulation/operations.go

```diff
@@ -0,0 +1,911 @@
+package simulation
+
+import (
+	"math/rand"
+	"strings"
+
+	"fmt"
+
+	"github.com/cosmos/cosmos-sdk/baseapp"
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/cosmos/cosmos-sdk/simapp/helpers"
+	simappparams "github.com/cosmos/cosmos-sdk/simapp/params"
+	"github.com/cosmos/cosmos-sdk/x/simulation"
+	gogotypes "github.com/gogo/protobuf/types"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	simtypes "github.com/cosmos/cosmos-sdk/types/simulation"
+	regentypes "github.com/regen-network/regen-ledger/types"
+	"github.com/regen-network/regen-ledger/x/group"
+	"github.com/regen-network/regen-ledger/x/group/exported"
+)
+
+// Simulation operation weights constants
+const (
+	OpMsgCreateGroupRequest               = "op_weight_msg_create_group"
+	OpMsgUpdateGroupAdminRequest          = "op_weight_msg_update_group_admin"
+	OpMsgUpdateGroupMetadata              = "op_wieght_msg_update_group_metadata"
+	OpMsgUpdateGroupMembers               = "op_weight_msg_update_group_members"
+	OpMsgCreateGroupAccountRequest        = "op_weight_msg_create_group_account"
+	OpMsgUpdateGroupAccountAdmin          = "op_weight_msg_update_group_account_admin"
+	OpMsgUpdateGroupAccountDecisionPolicy = "op_weight_msg_update_group_account_decision_policy"
+	OpMsgUpdateGroupAccountMetaData       = "op_weight_msg_update_group_account_metadata"
+	OpMsgCreateProposal                   = "op_weight_msg_create_proposal"
+	OpMsgVote                             = "op_weight_msg_vote"
+	OpMsgExec                             = "ops_weight_msg_exec"
+)
+
+//  If update group or group account txn's executed, `SimulateMsgVote` & `SimulateMsgExec` txn's returns `noOp`.
+//  That's why we have less weight for update group & group-account txn's.
+const (
+	WeightCreateGroup    = 150
+	WeightCreateProposal = 100
+	WeightUpdateGroup    = 5
+	WeightGroupMember    = 40
```

Comment:
We should declare weights for all the sdk messages. The SDK style of doing this is `DefaultWeightMsgMultiSend`, `DefaultWeightMsgDelegate` etc. Also, don't see a reason for keeping `150` for `WeightCreateGroup`. May be it can be just `100`. All sdk weights have either 100 or lesser. May be we should follow the same